misc/scripts/install-local.sh

```diff
@@ -571,9 +572,10 @@ DIR="$PWD"
 homedir="$(eval echo ~"$PACSTALL_USER")"
 export homedir
 
-pacfile=$(readlink -f "$PACKAGE".pacscript)
+sudo mv "${PACKAGE}.pacscript" /tmp
```

Comment:
Use `cp` instead. We don't want to move an user-made pacscript out of their workspace.